packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -82,6 +90,24 @@ export const AccountHeader = (props: AccountHeaderProps) => {
 
   return (
     <Box w='100%'>
+      {backButtonText ? (
+        <Box
+          css={{
+            position: 'absolute',
+            top: spacing.xl,
+            left: spacing.xl,
+            zIndex: 2
+          }}
+        >
+          <PlainButton
+            iconLeft={IconArrowLeft}
+            variant='inverted'
+            onClick={history.goBack}
```

Comment:
I don't love just doing `goBack` here but it should be fine the vast majority of the time. 
Right now there's technically a valid path where the user hits the X and their "back" is trending but its pretty rare and I think we may need to take away the X entirely on genres/artists tbh

Reply:
oh interesting, I didn't know about that flow. Is this worth fixing now, or waiting until the X button removal is decided?

Reply:
I think we just need to remove the X actually, the designs don't have it at all on those pages. Probably worth just double checking with design to be sure but yea
Probably fine to just leave the `goBack` here in this case

Reply:
yea looks like the x is removed after account finalization page